boards/PDM/Inc/Io/Io_Aux1Aux2Efuse.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include <assert.h>
+#include <math.h>
+#include "App_Efuse.h"
+#include "Io_Efuse.h"
+#include "Io_CurrentSense.h"
+
+/**
+ * Initialize the Aux1/Aux2 Efuse
+ * @param hspi Handle to the SPI peripheral used for the Efuse
+ */
+void Io_Aux1Aux2Efuse_Init(SPI_HandleTypeDef *hspi);
+
+/**
+ * Enable the Aux 1 channel output
+ */
+void Io_Aux1Aux2Efuse_Aux1Enable(void);
+
+/**
+ * Disable the Aux 1 channel output
+ */
+void Io_Aux1Aux2Efuse_Aux1Disable(void);
+
+/**
+ * Enable the Aux 2 channel output
+ */
+void Io_Aux1Aux2Efuse_Aux2Enable(void);
+
+/**
+ * Disable the Aux 2 channel output
+ */
+void Io_Aux1Aux2Efuse_Aux1Disable(void);
+
+/**
+ * Get the value of the Aux 1 and Aux 2 Efuse's Status Register over SPI
+ * @param status Pointer to the Aux1 and Aux2 channel's status
+ * @return EXIT_CODE_OK if the read was successful,
+ *         else EXIT_CODE_TIMEOUT if the SPI read timed-out
+ */
+ExitCode Io_Aux1Aux2Efuse_GetStatus(enum Efuse_Status *status);
+
+/**
+ * Get the value of the Aux 1 Fault Register over SPI
+ * @note Reading from the fault register reset's non-latchable faults
+ * @param fault Pointer to the Aux 1 fault status
+ * @return EXIT_CODE_OK if the read was successful,
+ *         else EXIT_CODE_TIMEOUT if the SPI read timed-out
+ */
+ExitCode Io_Aux1Aux2Efuse_GetAux1Faults(enum Efuse_Fault *fault);
+
+/**
+ * Get the value of the Aux 2 Fault Register over SPI
+ * @note Reading from the fault register reset's non-latchable faults
+ * @param fault Pointer to the Aux 2 fault status
+ * @return EXIT_CODE_OK if the read was successful,
+ *         else EXIT_CODE_TIMEOUT if the SPI read timed-out
+ */
+ExitCode Io_Aux1Aux2Efuse_GetAux2Faults(enum Efuse_Fault *fault);
+
+/**
+ * Check if the Efuse's Aux1 or Aux2 channel enters fault mode
```

Comment:
I think this was my bad. Introduced this change when I thought the Aux1 and Aux2 were separate efuses

```suggestion
 * Check if the Aux1Aux2 Efuse enters fault mode
```

Reply:
For the remaining comments we should also refer to the efuse as AuxAux2 efuse

Reply:
I think it's still correct since Aux1 is channel 0 on the efuse and Aux2 is channel1 🤔